client/webserver/webserver.go

```diff
@@ -32,25 +32,28 @@ import (
 )
 
 const (
-	// rpcTimeoutSeconds is the number of seconds a connection to the
-	// RPC server is allowed to stay open without authenticating before it
-	// is closed.
-	rpcTimeoutSeconds = 10
+	// httpConnTimeoutSeconds is the maximum number of seconds allowed for reading
+	// an http request or writing the response, beyond which the http connection is
+	// terminated.
+	httpConnTimeoutSeconds = 10
 	// darkModeCK is the cookie key for dark mode.
 	darkModeCK = "darkMode"
 	// authCK is the authorization token cookie key.
 	authCK = "dexauth"
-	// authCV is the authorization middleware contextual value key.
-	authCV = "authctx"
 )
 
+type contextKey string
+
```

Comment:
Why does this need its own type?

Reply:
`context.WithValue` discourages using built-in types. https://golang.org/pkg/context/#WithValue

I'll add a comment.

Reply:
Awesome.